src/GuBPI.fsproj

```diff
@@ -18,7 +18,7 @@
     <Compile Include="Typing.fs" />
     <Compile Include="ConstraintGeneration.fs" />
     <Compile Include="ConstraintSolver.fs" />
-    <Compile Include="VinciSystemCall.fs" />
+    <Compile Include="SystemCall.fs" />
```

Comment:
Is there a reason for the renaming? I find system call confusing. Aren't we just running vinci as a subprocess? I would find the name "Vinci.fs" better, if my reasoning is correct.